app/uk/gov/hmrc/apisubscriptionfields/repository/SubscriptionFieldsRepository.scala

```diff
@@ -78,25 +76,27 @@ class SubscriptionFieldsMongoRepository @Inject()(mongoDbProvider: MongoDbProvid
   )
 
   override def save(subscription: SubscriptionFields): Future[Boolean] = {
-    collection.update(selector = selectorForSubscriptionFields(subscription), update = subscription, upsert = true).map {
-      updateWriteResult => handleSaveError(updateWriteResult, s"Could not save subscription fields: $subscription",
-        updateWriteResult.upserted.nonEmpty
-      )
-    }
+    save(subscription, selectorForSubscriptionFields(subscription))
   }
 
   override def fetchByClientId(clientId: String): Future[List[SubscriptionFields]] = {
```

Comment:
Here `clientId` is a String
But below we use the type `ClientId`
Could you use the same type `ClientId` please?
We might review this later, but for now let's be consistent in the code

Reply:
done